src/main/java/ita/softserve/course_evaluation/controller/PermissionController.java

```diff
@@ -0,0 +1,53 @@
+package ita.softserve.course_evaluation.controller;
+
+import ita.softserve.course_evaluation.dto.PermissionDto;
+import ita.softserve.course_evaluation.entity.Permission;
+import ita.softserve.course_evaluation.service.PermissionService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
please, do not use wildcard import
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports